src/goto-programs/goto_program_template.h

```diff
@@ -386,6 +386,14 @@ class goto_program_templatet
     return t;
   }
 
+  static const irep_idt get_function_id(const_targett l)
+  {
+    while(!l->is_end_function())
+      ++l;
+
+    return l->function;
+  }
+
```

Comment:
Didn't we just remove this method?

Reply:
Indeed 5df3fca9cfe7b9732f0d87d1a0eaf740194bab87 removes this method. It seems that since by f443b1815e34bbec2b51fe14d732ba41d92010b7 cbmc does not do partial inlining anymore, the function field of an instruction is now guaranteed to contain the id of the function it is in. I'll remove the method again.

(In goto-instrument there are still a few cases where the function field is not adjusted when the inliner is called explicitely, but we can fix this in the future.)